Tests/ComposableArchitectureTests/EffectRunTests.swift

```diff
@@ -42,35 +41,37 @@ final class EffectRunTests: XCTestCase {
     await store.receive(.response)
   }
 
-  func testRunUnhandledFailure() async {
-    XCTExpectFailure(nil, enabled: nil, strict: nil) {
-      $0.compactDescription == """
-        An 'Effect.run' returned from "ComposableArchitectureTests/EffectRunTests.swift:62" threw \
-        an unhandled error. …
+  #if DEBUG
+    func testRunUnhandledFailure() async {
```

Comment:
None of the runtime warnings / test failure tests can run in release, so maybe we just move all such tests to their own files eventually.